src/components/User/AccountSettings/index.tsx

```
@@ -1,45 +1,38 @@
-  gender: 'other',
-}
 
-const AccountSettings = ({ userData }: { userData: User }) => {
+const AccountSettings = ({ userData, className }: { userData: User; className?: string }) => {
   const {
     control,
     handleSubmit,
-    formState: { errors },
-    setValue,
-    watch,
+    formState: {  },
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)